src/main/scala/codacy/pylint/Pylint.scala

```diff
@@ -36,24 +45,90 @@ object Pylint extends Tool {
 
   private val msgTemplate = "{path}###{line}###{msg}###{msg_id}"
 
-  private def commandFor(path: Path, conf: Option[Seq[PatternDef]], files: Option[Set[Path]])(implicit spec: Spec): Try[Seq[String]] = {
+  private def collectFiles(files: Option[Set[Path]], path: Path) = {
+    files.collect { case files if files.nonEmpty => files.map(_.toString) }.getOrElse {
+      //if files is empty, let the classification script to find them.
+      Seq(path.toString)
+    }
+  }
+
+  def printToFile(file: java.io.File, text: String) = {
+      val printer = new java.io.PrintWriter(file)
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Consider using util.Try instead of a try-catch block](https://www.codacy.com/app/Codacy/codacy-pylint/file/1728033956/issues/source?bid=2829823&fileBranchId=3054541#l57)